File: adventofcode-module/src/test/java/nl/mvdr/adventofcode/adventofcode2024/day08/Part2Test.java
package nl.mvdr.adventofcode.adventofcode2024.day08;

import java.util.stream.Stream;

import org.junit.jupiter.params.provider.Arguments;

import nl.mvdr.adventofcode.solver.SolverTest;

public class Part2Test extends SolverTest<Part2> {

    public Part2Test() {
        super(Part2.class);
    }
    
    static Stream<Arguments> testSolution() {
        return Stream.of(
                Arguments.of("34", "example-day08-2024-3.txt"),
                Arguments.of("9", "example-day08-2024-4.txt"));
    }
}

File: adventofcode-2025/src/test/kotlin/nl/mvdr/adventofcode/adventofcode2025/day08/Part2Test.kt
package nl.mvdr.adventofcode.adventofcode2025.day08

import nl.mvdr.adventofcode.solver.FunctionSolverTest
import org.junit.jupiter.params.provider.Arguments

class Part2Test: FunctionSolverTest<Long>(::solvePart2) {
    companion object {
        @JvmStatic
        fun testSolution(): List<Arguments> {
            return listOf(
                Arguments.of("25272", "example-day08.txt")
            )
        }
    }
}